### PR TITLE
Notify Things about Bridge status changes.

### PR DESCRIPTION
**Pull request by sbussweiler**
Signed-off-by: Stefan Bussweiler <s.bussweiler@external.telekom.de>

### CODE REVIEW

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java [hunk 1]
```diff
@@ -189,8 +189,10 @@ public void statusUpdated(final Thing thing, ThingStatusInfo thingStatus) {
                 if (oldStatus == ThingStatus.INITIALIZING && isInitialized(thing)) {
                     notifyThingsAboutBridgeInitialization(bridge);
                 }
-                // notify all child-things that bridge is ONLINE/OFFLINE
-                notifyThingsAboutBridgeStatusUpdate(thingStatus, bridge);
+                // update status of child-things
+                updateThingStatus(thingStatus, bridge);
+                // notify child-things about bridge status change, if bridge status is ONLINE/OFFLINE
+                notifyThingsAboutBridgeStatusChange(thingStatus, bridge);
```
Comment (kaikreuzer) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java]: Do you filter anywhere that this is really a CHANGE and not just an UPDATE?

**Comment by kaikreuzer** [comment]
Thanks, apart from the one comment this looks good to me.

**Comment by sbussweiler** [comment]
Updated PR. Notify child things for a real bridge status change, not just an update. Cf. [ThingManager](https://github.com/eclipse/smarthome/pull/1142/files#diff-24d71ac793f6de6c8557ea06292497ddR195).

**Comment by kaikreuzer** [comment]
thanks. @maggu2810, ok for me to be merged.

**Pull request merged**